### ISSUE TITLE
Public datasets are not accessible by logged in user

### ISSUE THREAD

**Issue by hokinus** [NSSAC/CINES#351]
Srini reported that the dataset https://net.science/files/resources/datasets/PET_Prize_PandemicForecasting/ was not available to him, when he was logged in:
![4dff1b92-126e-44af-9826-50512854229e](https://github.com/NSSAC/CINES/assets/4824691/b80777b0-9f8d-4919-8159-d27aa36af40a)

It worked using anonymous access

**Comment by dmachi** [comment]
Yes, this is that problem from a month or so ago.  It is fixed, but not deployed to production yet.  That push to production is expected within the next few days.
